package.json

```diff
@@ -170,6 +170,7 @@
     "graphql": "^0.13.2",
     "graphql-redis-subscriptions": "^1.2.0",
     "graphql-relay": "^0.5.2",
+    "hoist-non-react-statics": "^2.5.0",
```

Comment:
react-dnd uses ^2.3.0, but we need ^2.5.0, so this tricks it into using the right version